apps/theming-designer/src/components/FabricPalette.tsx

```diff
@@ -73,66 +49,264 @@ export const FabricPalette: React.StatelessComponent<IFabricPaletteProps> = (pro
         </thead>
         <tbody>
           <tr>
-            <Text as="td">{fabricSlotWidget(FabricSlots.themeDarker)}</Text>
-            <Text as="td">{props.themeRules![FabricSlots[FabricSlots.themeDarker]].color!.str}</Text>
-            <Text as="td">{fabricSlotWidget(FabricSlots.black)}</Text>
-            <Text as="td">{props.themeRules![FabricSlots[FabricSlots.black]].color!.str}</Text>
-            <Text as="td">{fabricSlotWidget(FabricSlots.neutralTertiaryAlt)}</Text>
-            <Text as="td">{props.themeRules![FabricSlots[FabricSlots.neutralTertiaryAlt]].color!.str}</Text>
+            <Text as="td">
+              <FabricSlotWidget
+                themeRules={props.themeRules!}
+                slot={FabricSlots.themeDarker}
+                onFabricPaletteColorChange={props.onFabricPaletteColorChange}
+                directionalHint={DirectionalHint.leftCenter}
+              />
+            </Text>
+            <Text as="td">
+              <div>{props.themeRules[FabricSlots[FabricSlots.themeDarker]].color.str}</div>
```

Comment:
Nit, you likely don't need these wrapper `DIV` elements throughout the table cells since a string value inside of a `<td>` is acceptable markup. See the following as an example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table